<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>247</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>CONNEXION</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../res/ressources.qrc">
         <normaloff>:/icons/icons/account_circle.png</normaloff>:/icons/icons/account_circle.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="usernameIn"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../res/ressources.qrc">
         <normaloff>:/icons/icons/password.png</normaloff>:/icons/icons/password.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passwordIn"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ANNULER</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loginBtn">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>LOGIN</string>
       </property>
       <property name="icon">
        <iconset resource="../../../res/ressources.qrc">
         <normaloff>:/icons/icons/login.png</normaloff>:/icons/icons/login.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../res/ressources.qrc"/>
 </resources>
 <connections/>
</ui>
